.github/workflows/cleanup.yml

```diff
@@ -1,14 +1,38 @@
 name: cleanup
 
 on:
-  schedule:
-    - cron: '*/5 * * * *'
+  # Triggerred on every workflow run except on master
+  workflow_run:
+    workflows: [ "ci" ]
+    types:
+      - requested
+    branches-ignore: [ "master" ]
```

Comment:
use `[` or `-` lists consistently (i think we favor `-` lists, but i appreciate `[ .. ]` can be more useful for brevity, and so readability)